README.md

```diff
@@ -1,4 +1,38 @@
-# Python boilerplate for GitHub tasks
+# Distance class
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
+
+Every day you have morning running. You want to store your result. 
+To easy calculating, comparison and printing you result write class
+`Distance`.
+
+`Distance`'s constructor takes only
+one argument `km` and saves it to `self.km`.
+
+For `Distance` class you should implement such magic
+methods:
+```python
+__init__
+distance = Distance(20) # distance.km == 20
+__str__
+print(distance) # "Distance: 20 kilometers."
+__add__
+dist2 = distance + 10 # dist2 == 30
```

Comment:
Here I wanted not such operation, but next:
```python
day1 = Distance(10)
day2 = Distance(30)

two_days = day1 + day2  # two_days.km == 40
```

Reply:
But right now, I think, that it will be good, if possible both,
So we can add day1 to day2,
Or just add day1 to integer - both should work :)